src/Forms/MoveElementHandler.php

```diff
@@ -0,0 +1,143 @@
+<?php
+
+namespace DNADesign\Elemental\Forms;
+
+use DNADesign\Elemental\Extensions\ElementalPageExtension;
+use SilverStripe\CMS\Model\SiteTree;
+use SilverStripe\Core\Injector\Injectable;
+use SilverStripe\Forms\FieldList;
+use SilverStripe\Forms\Form;
+use SilverStripe\Forms\FormAction;
+use SilverStripe\Forms\HiddenField;
+use SilverStripe\Forms\LiteralField;
+use SilverStripe\Forms\TreeDropdownField;
+use SilverStripe\ORM\ValidationException;
+use SilverStripe\ORM\ValidationResult;
+
+class MoveElementHandler
+{
+    use Injectable;
+
+    /**
+     * Parent controller for this form
+     *
+     * @var Controller
+     */
+    protected $controller;
+
+    public function __construct($controller = null)
+    {
+        $this->controller = $controller;
+    }
+
+    public function Form($elementID)
+    {
+        $fields = FieldList::create([
+            LiteralField::create(
+                'MoveWarning',
+                '<p class="alert alert-info"><strong>Note</strong>: All published blocks will changed draft state once moved.</p>'
+            ),
+            HiddenField::create(
+                'ElementID',
+                null,
+                $elementID
+            ),
+            $pageField = TreeDropdownField::create(
```

Comment:
Limit to pages with elemental active

Reply:
Yep, this is done on L62

Reply:
I was able to move a block to a page without elemental